db/data_migrate/20180807095900_import_customers.rb

```diff
@@ -0,0 +1,20 @@
+require 'csv'
+require 'progress_bar'
+
+puts 'Reading CSV file...'
+csv = CSV.table(Rails.root.join('db', 'data_migrate', '20180807095900_customers.csv'))
+
+puts 'Importing customers...'
+bar = ProgressBar.new(csv.count)
+csv.each do |customer_row|
+  postcode = customer_row[:postcode] == 'XXXX' ? nil : customer_row[:postcode]
+
+  Customer.create!(
```

Comment:
How about making this able to be run more than once? (i.e. if the VPN dies in the middle of the script running)

Also, it's probably worth calling `strip` on all these input strings, and a lot of the other exports have included erroneous whitespace.

Reply:
updated.